modules/data-wrangling/index.qmd

```diff
@@ -69,6 +33,10 @@ Did you know you can also load specific library separately, for example, `librar
 * `count()`
 * `case_when()`
 
+:::{.callout-note}
+This is a very big topic, and we will not cover all of the useful functions in this chapter. Here is a cheat sheet to help you learn more about [dplyr](https://github.com/rstudio/cheatsheets/blob/main/data-transformation.pdf).
```

Comment:
This sounds apologetic - instead I think it could be more focused on what the section covers rather than what it doesn't and emphasise the cheat sheet.

For example:

```md
::: {.callout-note}
## Cheatsheet

This section covers the basic and commonly used dplyr functions for data manipulation. A more comprehensive summary of the functions in the dplyr package can be found in the [dplyr cheatsheet](https://github.com/rstudio/cheatsheets/blob/main/data-transformation.pdf).
:::
```